week3/assign1/src/components/StageThree.jsx

```diff
@@ -0,0 +1,93 @@
+import styled from "styled-components";
+import Button from "./Button";
+import { useState } from "react";
+
+const StageThree = (props) => {
+  const [state, setState] = useState(null);
+
+  const change = (nextstage) => {
+    props.changeStageValue(3,state);
+    props.changeStage(nextstage);
+  };
+
+  return (
+    <Wrapper>
+      <div id="progress">3 / 3</div>
+      <RadioContainer>
+        <RadioLabel selected={state === 0} onClick={() => setState(0)}>
+          국물O
+          <HiddenRadio
+            type="radio"
+            name="soup"
+            value="국물O"
+          />
+        </RadioLabel>
+        <RadioLabel selected={state === 1} onClick={() => setState(1)}>
+          국물X
+          <HiddenRadio
+            type="radio"
+            name="soup"
+            value="국물X"
+          />
+        </RadioLabel>
+      </RadioContainer>
+      <ButtonContainer>
+        <Button content="이전으로" onClick={() => change(2)} state={2}></Button>
+        <Button content="결과보기" onClick={() => change(4)} state={state}></Button>
+      </ButtonContainer>
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.div`
```

Comment:
이렇게 반복적으로 사용되는 스타일요소는 따로 빼서 필요에따라 import 해서 쓰면 편해!!
```suggestion
export const Wrapper = styled.div` ~~ `;

import { Wrapper } from ~~
```